.github/workflows/build-master.yml

```diff
@@ -0,0 +1,26 @@
+name: Build Master Branch
```

Comment:
```suggestion
name: Build Main Branch
```

Reply:
Fixed in 823fd0e51f8f8a460d5e83b3c1026d65520a7a16

Reply:
Fixed in 81c27da9fbdc7f8ceaaa80228a825f2afd8a13e7